utility/src/main/java/org/oppia/android/util/extensions/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+"""
+General purposes utilities to manage extensions
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "bundle_extensions",
+    srcs = [
+        "BundleExtensions.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:com_google_protobuf_protobuf-lite",
+    ],
+)
+
+kt_android_library(
+    name = "live_data_ext",
```

Comment:
Let's call it 'Extensions' for consistency. Please also rename ``LiveDataExt`` while here for consistency.

Reply:
Done